src/App.jsx

```diff
@@ -1,6 +1,6 @@
 import React, { Component } from 'react';
 import { Provider } from 'react-redux';
-import { ToastContainer } from 'react-toastify';
+import { ToastContainer, toast, Zoom } from 'react-toastify';
```

Comment:
@Akinmyde Good job on this implementation... What does `Zoom` do?

Reply:
> @Akinmyde Good job on this implementation... What does `Zoom` do?

Zoom is used for transition in the toast notification